src/sif/pedestriancost.cc

```diff
@@ -435,6 +435,9 @@ class PedestrianCost : public DynamicCost {
                                  const uint32_t idx) const override {
     // Cases with both time and penalty: country crossing, ferry, gate, toll booth
     sif::Cost c;
+    if (shortest_) {
+      return c; // shortest ignores any penalties in favor of path length
+    }
```

Comment:
not sure about this one @kevinkreiser.. from the description it sounds like it should be added here as well for ped part of multimodal costing?